<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html>
  <head>
    <META http-equiv="Content-Type" content="text/html; charset=utf-8">
    <title>xUnit.net Test Results - C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\bin\Debug\tests.xunit.1.1.DLL</title>
    <style type="text/css">
          body { font-family: Calibri, Verdana, Arial, sans-serif; background-color: White; color: Black; }
          h2,h3,h5 { margin: 0; padding: 0; }
          h2 { border-top: solid 1px #f0f5fa; padding-top: 0.5em; }
          h3 { font-weight: normal; }
          h5 { font-weight: normal; font-style: italic; margin-bottom: 0.75em; }
          pre { font-family: Consolas; font-size: 85%; margin: 0 0 0 1em; padding: 0; }
          .row, .altrow { padding: 0.1em 0.3em; }
          .row { background-color: #f0f5fa; }
          .altrow { background-color: #e1ebf4; }
          .success, .failure, .skipped { font-family: Arial Unicode MS; font-weight: normal; float: left; width: 1em; display: block; }
          .success { color: #0c0; }
          .failure { color: #c00; }
          .skipped { color: #cc0; }
          .timing { float: right; }
          .indent { margin: 0.25em 0 0.5em 2em; }
          .clickable { cursor: pointer; }
          .testcount { font-size: 85%; }
        </style><script language="javascript">
          function ToggleClass(id) {
            var elem = document.getElementById(id);
            if (elem.style.display == "none")
              elem.style.display = "block";
            else
              elem.style.display = "none";
          }
        </script></head>
  <body>
    <h3><b>Results for C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\bin\Debug\tests.xunit.1.1.DLL</b></h3>
    <div>
          Tests run: <a href="#all"><b>108</b></a>  
          Failures: <a href="#failures"><b>29</b></a>,
          Skipped: <a href="#skipped"><b>6</b></a>,
          Run time: <b>7.864s</b></div><br><h2><a name="failures"></a>Failed tests</h2>
    <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       Name contains spaces : System.NotImplementedException : The method or operation is not implemented.<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithDisplayNames.ShouldNotSeeThis_NameShouldContainSpaces() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithDisplayNames.cs:line 26</pre>
    </div>
    <div class="altrow"><span class="timing">0.139s</span><span class="failure">✘</span>
       NameComesFromAttribute : System.NotImplementedException : The method or operation is not implemented.<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithDisplayNames.ShouldNotSeeThis_NameShouldComeFromAttribute() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithDisplayNames.cs:line 18</pre>
    </div>
    <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.CausingFrameworkFailures+AmbiguousMethodNames.MethodsWithOverridesCannotBeUniquelyFoundByName : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.CausingFrameworkFailures+AmbiguousMethodNames.MethodsWithOverridesCannotBeUniquelyFoundByName : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="row"><span class="timing">0.017s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToAssert : Assert.Equal() Failure
Position: First difference is at position 0
Expected: this test
Actual:   should fail<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 29</pre>
    </div>
    <div class="altrow"><span class="timing">0.024s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToDebugAssert : Debug.Assert() Failure : message - supposed to fail
Detailed Message:
detailed message - supposed to fail<br clear="all"><pre>   at System.Diagnostics.TraceInternal.Fail(String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToDebugAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 36</pre>
    </div>
    <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToThrownException : System.Exception : this test should fail<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToThrownException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 20</pre>
    </div>
    <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToTraceAssert : Debug.Assert() Failure : message - supposed to fail
Detailed Message:
detailed message - supposed to fail<br clear="all"><pre>   at System.Diagnostics.TraceInternal.Fail(String message, String detailMessage)
   at System.Diagnostics.Trace.Assert(Boolean condition, String message, String detailMessage)
   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToTraceAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 43</pre>
    </div>
    <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("fail as well", 13) : Assert.Equal() Failure
Expected: 13
Actual:   12<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
    </div>
    <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("failing", 1) : Assert.Equal() Failure
Expected: 1
Actual:   7<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
    </div>
    <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("fails", 6) : Assert.Equal() Failure
Expected: 6
Actual:   5<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
    </div>
    <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest("failing", 1) : Assert.Equal() Failure
Expected: 1
Actual:   7<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 139</pre>
    </div>
    <div class="row"><span class="timing">0.011s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributesThrowBeforeAndAfterTest : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
    </div>
    <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributesThrowMultipleTimesAfterTests : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)

System.InvalidProgramException thrown: Second thrown exception
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAgainAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 330
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
    </div>
    <div class="row"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributeThrowsAfterTest : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
    </div>
    <div class="altrow"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributeThrowsBeforeTest : System.InvalidOperationException : Message from Before attribute<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingBeforeTestAttribute.Before(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 314
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="row"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInConstructor.AnotherTest : System.Reflection.TargetInvocationException : Exception has been thrown by the target of an invocation.
---- System.InvalidOperationException : Bad things happened in class constructor<br clear="all"><pre>   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean&amp; canBeCached, RuntimeMethodHandle&amp; ctor, Boolean&amp; bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean fillCache)
   at System.RuntimeType.CreateInstanceImpl(Boolean publicOnly, Boolean skipVisibilityChecks, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInConstructor..ctor() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 194</pre>
    </div>
    <div class="altrow"><span class="timing">0.074s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInConstructor.ClassThrowsInConstructor : System.Reflection.TargetInvocationException : Exception has been thrown by the target of an invocation.
---- System.InvalidOperationException : Bad things happened in class constructor<br clear="all"><pre>   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean&amp; canBeCached, RuntimeMethodHandle&amp; ctor, Boolean&amp; bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean fillCache)
   at System.RuntimeType.CreateInstanceImpl(Boolean publicOnly, Boolean skipVisibilityChecks, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInConstructor..ctor() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 194</pre>
    </div>
    <div class="row"><span class="timing">0.037s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInDispose.TestThrowsAssert : System.InvalidOperationException : Something bad happened in class Dispose method<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInDispose.Dispose() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 218
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInDispose.WorkingTest : System.InvalidOperationException : Something bad happened in class Dispose method<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInDispose.Dispose() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 218
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="row"><span class="timing">0.005s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureDispose.TestThrowsAssert : Assert.Equal() Failure
Expected: 1
Actual:   134<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInFixtureDispose.TestThrowsAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 301</pre>
    </div>
    <div class="altrow"><span class="timing">0.059s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.InvalidFactMethod.TooManyParametersForFact : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
    </div>
    <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException : System.InvalidProgramException : Invalid program exception message
---- System.InvalidOperationException : Invalid operation exception message<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 73
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallAnotherMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 111
   at tests.xunit.ExpectedToFail.NestedExceptions.CallSomeMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 106
   at tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 67</pre>
    </div>
    <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions : System.InvalidProgramException : Invalid program exception message
---- System.IO.InvalidDataException : Invalid data exception message
-------- System.InvalidOperationException : Invalid operation exception message<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 102
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallOneMoreMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 124
   at tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 96
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallAnotherMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 111
   at tests.xunit.ExpectedToFail.NestedExceptions.CallSomeMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 106
   at tests.xunit.ExpectedToFail.NestedExceptions.CallOneMoreMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 118</pre>
    </div>
    <div class="row"><span class="timing">0.005s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestHasNameTrait : System.NotImplementedException : What should this do?<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestHasNameTrait() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithTraits.cs:line 13</pre>
    </div>
    <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestMethodHasCategoryTrait : System.NotImplementedException : Categories not yet supported<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestMethodHasCategoryTrait() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithTraits.cs:line 19</pre>
    </div>
    <div class="row"><span class="timing">0.106s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.TimedOutTests.TestShouldTimeOut : Test execution time exceeded: 100ms<br clear="all"></div><br><h2><a name="failures"></a>Failed fixtures</h2><span class="failure">✘</span>tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureConstructor : System.InvalidOperationException : Something bad happened in the fixture constructor<br clear="all">
    Stack Trace:<br><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.FixtureThatThrowsInConstructor..ctor() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 243</pre><span class="failure">✘</span>tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureDispose : System.InvalidOperationException : Something bad happened in the fixture Dispose method<br clear="all">
    Stack Trace:<br><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.FixtureThatThrowsInDispose.Dispose() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 258
   at Xunit.Sdk.TestClassCommand.ClassFinish()</pre><br><h2><a name="skipped"></a>Skipped tests</h2>
    <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesDebugTraceOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    <div class="altrow"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceErrorOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceInformationOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    <div class="altrow"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceWarningOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    <div class="altrow"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.SkippedTests.SkippedTestMethod : This is the skip reason<br clear="all"></div><br><h2><a name="all"></a>All tests</h2>
    <h5>Click test class name to expand/collapse test details</h5>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class1')" ondblclick="ToggleClass('class1')"><span class="success">✔</span>
         tests.xunit.CapturesOutput.CapturesDebugTrace
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class1">
      <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesDebugTraceOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
      <div class="altrow"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceErrorOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
      <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceInformationOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
      <div class="altrow"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
      <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.CapturesOutput.CapturesDebugTrace.CapturesTraceWarningOutput : xunit does not support capturing output from System.Diagnostics.Trace<br clear="all"></div>
    </div>
    <h3><span class="timing">0.002s</span><span class="clickable" onclick="ToggleClass('class2')" ondblclick="ToggleClass('class2')"><span class="success">✔</span>
         tests.xunit.CapturesOutput.CapturesStandardOutput
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class2">
      <div class="row"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.CapturesOutput.CapturesStandardOutput.CapturesStdErr<br clear="all"></div>
      <div class="altrow"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.CapturesOutput.CapturesStandardOutput.CapturesStdOut<br clear="all"></div>
    </div>
    <h3><span class="timing">0.001s</span><span class="clickable" onclick="ToggleClass('class3')" ondblclick="ToggleClass('class3')"><span class="success">✔</span>
         tests.xunit.CustomAttributes.CustomFactAttribute
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class3">
      <div class="row"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomFactAttribute.MyTestMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">0.001s</span><span class="clickable" onclick="ToggleClass('class4')" ondblclick="ToggleClass('class4')"><span class="success">✔</span>
         tests.xunit.CustomAttributes.CustomTheoryAttribute
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class4">
      <div class="row"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomTheoryAttribute.MyTestMethod<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomTheoryAttribute.MyTestMethod<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomTheoryAttribute.MyTestMethod<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomTheoryAttribute.MyTestMethod<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.CustomAttributes.CustomTheoryAttribute.MyTestMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">2.005s</span><span class="clickable" onclick="ToggleClass('class5')" ondblclick="ToggleClass('class5')"><span class="success">✔</span>
         tests.xunit.DemonstrateRandomRunningOrder
         <span class="testcount">(20 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class5">
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test01<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test02<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test03<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test04<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test05<br clear="all"></div>
      <div class="altrow"><span class="timing">0.102s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test06<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test07<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test08<br clear="all"></div>
      <div class="row"><span class="timing">0.101s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test09<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test10<br clear="all"></div>
      <div class="row"><span class="timing">0.101s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test11<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test12<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test13<br clear="all"></div>
      <div class="altrow"><span class="timing">0.101s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test14<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test15<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test16<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test17<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test18<br clear="all"></div>
      <div class="row"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test19<br clear="all"></div>
      <div class="altrow"><span class="timing">0.100s</span><span class="success">✔</span>
       tests.xunit.DemonstrateRandomRunningOrder.Test20<br clear="all"></div>
    </div>
    <h3><span class="timing">0.003s</span><span class="clickable" onclick="ToggleClass('class6')" ondblclick="ToggleClass('class6')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.CausingFrameworkFailures+AmbiguousMethodNames
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class6">
      <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.CausingFrameworkFailures+AmbiguousMethodNames.MethodsWithOverridesCannotBeUniquelyFoundByName : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
      <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.CausingFrameworkFailures+AmbiguousMethodNames.MethodsWithOverridesCannotBeUniquelyFoundByName : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
    </div>
    <h3><span class="timing">0.042s</span><span class="clickable" onclick="ToggleClass('class7')" ondblclick="ToggleClass('class7')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailingTests
         <span class="testcount">(4 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class7">
      <div class="row"><span class="timing">0.017s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToAssert : Assert.Equal() Failure
Position: First difference is at position 0
Expected: this test
Actual:   should fail<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 29</pre>
      </div>
      <div class="altrow"><span class="timing">0.024s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToDebugAssert : Debug.Assert() Failure : message - supposed to fail
Detailed Message:
detailed message - supposed to fail<br clear="all"><pre>   at System.Diagnostics.TraceInternal.Fail(String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToDebugAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 36</pre>
      </div>
      <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToThrownException : System.Exception : this test should fail<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToThrownException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 20</pre>
      </div>
      <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTests.FailsDueToTraceAssert : Debug.Assert() Failure : message - supposed to fail
Detailed Message:
detailed message - supposed to fail<br clear="all"><pre>   at System.Diagnostics.TraceInternal.Fail(String message, String detailMessage)
   at System.Diagnostics.Trace.Assert(Boolean condition, String message, String detailMessage)
   at tests.xunit.ExpectedToFail.FailingTests.FailsDueToTraceAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 43</pre>
      </div>
    </div>
    <h3><span class="timing">0.002s</span><span class="clickable" onclick="ToggleClass('class8')" ondblclick="ToggleClass('class8')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailingTheoryTests
         <span class="testcount">(6 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class8">
      <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("fail as well", 13) : Assert.Equal() Failure
Expected: 13
Actual:   12<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
      </div>
      <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("failing", 1) : Assert.Equal() Failure
Expected: 1
Actual:   7<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
      </div>
      <div class="row"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest("fails", 6) : Assert.Equal() Failure
Expected: 6
Actual:   5<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.MultipleFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 152</pre>
      </div>
      <div class="altrow"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest("failing", 1) : Assert.Equal() Failure
Expected: 1
Actual:   7<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest(String value, Int32 expectedLength) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 139</pre>
      </div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest("works as well", 13)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.ExpectedToFail.FailingTheoryTests.OneFailingTheoryTest("works", 5)<br clear="all"></div>
    </div>
    <h3><span class="timing">0.014s</span><span class="clickable" onclick="ToggleClass('class9')" ondblclick="ToggleClass('class9')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute
         <span class="testcount">(4 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class9">
      <div class="row"><span class="timing">0.011s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributesThrowBeforeAndAfterTest : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
      </div>
      <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributesThrowMultipleTimesAfterTests : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)

System.InvalidProgramException thrown: Second thrown exception
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAgainAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 330
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
      </div>
      <div class="row"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributeThrowsAfterTest : Xunit.Sdk.AfterTestException : One or more exceptions were thrown from After methods during test cleanup<br clear="all"><pre>System.InvalidOperationException thrown: Message from After attribute
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingAfterTestAttribute.After(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 322
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)</pre>
      </div>
      <div class="altrow"><span class="timing">0.000s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInBeforeAfterAttribute.AttributeThrowsBeforeTest : System.InvalidOperationException : Message from Before attribute<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowingBeforeTestAttribute.Before(MethodInfo methodUnderTest) in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 314
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
    </div>
    <h3><span class="timing">0.075s</span><span class="clickable" onclick="ToggleClass('class10')" ondblclick="ToggleClass('class10')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInConstructor
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class10">
      <div class="row"><span class="timing">0.001s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInConstructor.AnotherTest : System.Reflection.TargetInvocationException : Exception has been thrown by the target of an invocation.
---- System.InvalidOperationException : Bad things happened in class constructor<br clear="all"><pre>   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean&amp; canBeCached, RuntimeMethodHandle&amp; ctor, Boolean&amp; bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean fillCache)
   at System.RuntimeType.CreateInstanceImpl(Boolean publicOnly, Boolean skipVisibilityChecks, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInConstructor..ctor() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 194</pre>
      </div>
      <div class="altrow"><span class="timing">0.074s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInConstructor.ClassThrowsInConstructor : System.Reflection.TargetInvocationException : Exception has been thrown by the target of an invocation.
---- System.InvalidOperationException : Bad things happened in class constructor<br clear="all"><pre>   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean&amp; canBeCached, RuntimeMethodHandle&amp; ctor, Boolean&amp; bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean fillCache)
   at System.RuntimeType.CreateInstanceImpl(Boolean publicOnly, Boolean skipVisibilityChecks, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInConstructor..ctor() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 194</pre>
      </div>
    </div>
    <h3><span class="timing">0.039s</span><span class="clickable" onclick="ToggleClass('class11')" ondblclick="ToggleClass('class11')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInDispose
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class11">
      <div class="row"><span class="timing">0.037s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInDispose.TestThrowsAssert : System.InvalidOperationException : Something bad happened in class Dispose method<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInDispose.Dispose() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 218
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
      <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInDispose.WorkingTest : System.InvalidOperationException : Something bad happened in class Dispose method<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInDispose.Dispose() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 218
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
    </div>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class12')" ondblclick="ToggleClass('class12')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureConstructor
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" id="class12"></div>
    <h3><span class="timing">0.006s</span><span class="clickable" onclick="ToggleClass('class13')" ondblclick="ToggleClass('class13')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureDispose
         <span class="testcount">(3 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class13">
      <div class="row"><span class="timing">0.005s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureDispose.TestThrowsAssert : Assert.Equal() Failure
Expected: 1
Actual:   134<br clear="all"><pre>   at tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode.ThrowsInFixtureDispose.TestThrowsAssert() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 301</pre>
      </div>
      <div class="altrow"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.ExpectedToFail.FailuresInNonTestMethodCode+ThrowsInFixtureDispose.WorkingTest<br clear="all"></div>
    </div>
    <h3><span class="timing">0.059s</span><span class="clickable" onclick="ToggleClass('class14')" ondblclick="ToggleClass('class14')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.InvalidFactMethod
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" id="class14">
      <div class="row"><span class="timing">0.059s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.InvalidFactMethod.TooManyParametersForFact : System.Reflection.TargetParameterCountException : Parameter count mismatch.<br clear="all"><pre>   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture, Boolean skipVisibilityChecks)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Xunit.Sdk.TestCommand.Execute(Object testClass)
   at Xunit.Sdk.FixtureCommand.Execute(Object testClass)
   at Xunit.Sdk.BeforeAfterCommand.Execute(Object testClass)
   at Xunit.Sdk.LifetimeCommand.Execute(Object testClass)
   at Xunit.Sdk.ExceptionWrapperCommand.&lt;&gt;c__DisplayClass1.&lt;Execute&gt;b__0()
   at Xunit.Sdk.ExceptionWrapperCommand.RunAndCatch(ExecutionDelegate delegate)</pre>
      </div>
    </div>
    <h3><span class="timing">0.004s</span><span class="clickable" onclick="ToggleClass('class15')" ondblclick="ToggleClass('class15')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.NestedExceptions
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class15">
      <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException : System.InvalidProgramException : Invalid program exception message
---- System.InvalidOperationException : Invalid operation exception message<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 73
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallAnotherMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 111
   at tests.xunit.ExpectedToFail.NestedExceptions.CallSomeMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 106
   at tests.xunit.ExpectedToFail.NestedExceptions.HasInnerException() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 67</pre>
      </div>
      <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions : System.InvalidProgramException : Invalid program exception message
---- System.IO.InvalidDataException : Invalid data exception message
-------- System.InvalidOperationException : Invalid operation exception message<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 102
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallOneMoreMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 124
   at tests.xunit.ExpectedToFail.NestedExceptions.HasThreeInnerExceptions() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 96
----- Inner Stack Trace -----
   at tests.xunit.ExpectedToFail.NestedExceptions.CallAnotherMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 111
   at tests.xunit.ExpectedToFail.NestedExceptions.CallSomeMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 106
   at tests.xunit.ExpectedToFail.NestedExceptions.CallOneMoreMethod() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\FailingTests.cs:line 118</pre>
      </div>
    </div>
    <h3><span class="timing">0.141s</span><span class="clickable" onclick="ToggleClass('class16')" ondblclick="ToggleClass('class16')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithDisplayNames
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class16">
      <div class="row"><span class="timing">0.002s</span><span class="failure">✘</span>
       Name contains spaces : System.NotImplementedException : The method or operation is not implemented.<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithDisplayNames.ShouldNotSeeThis_NameShouldContainSpaces() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithDisplayNames.cs:line 26</pre>
      </div>
      <div class="altrow"><span class="timing">0.139s</span><span class="failure">✘</span>
       NameComesFromAttribute : System.NotImplementedException : The method or operation is not implemented.<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithDisplayNames.ShouldNotSeeThis_NameShouldComeFromAttribute() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithDisplayNames.cs:line 18</pre>
      </div>
    </div>
    <h3><span class="timing">0.007s</span><span class="clickable" onclick="ToggleClass('class17')" ondblclick="ToggleClass('class17')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" id="class17">
      <div class="row"><span class="timing">0.005s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestHasNameTrait : System.NotImplementedException : What should this do?<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestHasNameTrait() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithTraits.cs:line 13</pre>
      </div>
      <div class="altrow"><span class="timing">0.002s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestMethodHasCategoryTrait : System.NotImplementedException : Categories not yet supported<br clear="all"><pre>   at tests.xunit.ExpectedToFail.NotYetImplemented.TestsWithTraits.TestMethodHasCategoryTrait() in C:\Users\matt\Projects\OpenSource\xunitcontrib\resharper\tests.xunitcontrib.runner.resharper.manual\tests.xunit.1.1\TestsWithTraits.cs:line 19</pre>
      </div>
    </div>
    <h3><span class="timing">0.106s</span><span class="clickable" onclick="ToggleClass('class18')" ondblclick="ToggleClass('class18')"><span class="failure">✘</span>
         tests.xunit.ExpectedToFail.TimedOutTests
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" id="class18">
      <div class="row"><span class="timing">0.106s</span><span class="failure">✘</span>
       tests.xunit.ExpectedToFail.TimedOutTests.TestShouldTimeOut : Test execution time exceeded: 100ms<br clear="all"></div>
    </div>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class19')" ondblclick="ToggleClass('class19')"><span class="success">✔</span>
         tests.xunit.MixedMethodVisibility.PublicClass
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class19">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.MixedMethodVisibility.PublicClass.PublicMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">0.325s</span><span class="clickable" onclick="ToggleClass('class20')" ondblclick="ToggleClass('class20')"><span class="success">✔</span>
         tests.xunit.ReadsFromConfigFile.ReadsFromConfigFile
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class20">
      <div class="row"><span class="timing">0.325s</span><span class="success">✔</span>
       tests.xunit.ReadsFromConfigFile.ReadsFromConfigFile.ReadsAppSettings<br clear="all"></div>
    </div>
    <h3><span class="timing">0.012s</span><span class="clickable" onclick="ToggleClass('class21')" ondblclick="ToggleClass('class21')"><span class="success">✔</span>
         tests.xunit.SkippedTests
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class21">
      <div class="row"><span class="timing">Skipped</span><span class="skipped">❢</span>
       tests.xunit.SkippedTests.SkippedTestMethod : This is the skip reason<br clear="all"></div>
    </div>
    <h3><span class="timing">5.003s</span><span class="clickable" onclick="ToggleClass('class22')" ondblclick="ToggleClass('class22')"><span class="success">✔</span>
         tests.xunit.SuccessfulTheoryTests.TheoryTests
         <span class="testcount">(14 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class22">
      <div class="row"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(1)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(10)<br clear="all"></div>
      <div class="row"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(2)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(3)<br clear="all"></div>
      <div class="row"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(4)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(5)<br clear="all"></div>
      <div class="row"><span class="timing">0.500s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(6)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(7)<br clear="all"></div>
      <div class="row"><span class="timing">0.499s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(8)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.500s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.DemonstrateTheoryTestRandomOrdering(9)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.SuccessfulMultipleLineTheoryTest("cheese", 6)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.011s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.SuccessfulMultipleLineTheoryTest("hello world", 11)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.SuccessfulMultipleLineTheoryTest("hello", 5)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.SuccessfulTheoryTests.TheoryTests.SuccessfulSingleLineTheoryTest("hello", 5)<br clear="all"></div>
    </div>
    <h3><span class="timing">0.002s</span><span class="clickable" onclick="ToggleClass('class23')" ondblclick="ToggleClass('class23')"><span class="success">✔</span>
         tests.xunit.TestClassLifecycle.TestClassLifecycle
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class23">
      <div class="row"><span class="timing">0.002s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TestClassLifecycle.Test01<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TestClassLifecycle.Test02<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TestClassLifecycle.Test03<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TestClassLifecycle.Test04<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TestClassLifecycle.Test05<br clear="all"></div>
    </div>
    <h3><span class="timing">0.003s</span><span class="clickable" onclick="ToggleClass('class24')" ondblclick="ToggleClass('class24')"><span class="success">✔</span>
         tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class24">
      <div class="row"><span class="timing">0.003s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle.AllTests(1)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle.AllTests(2)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle.AllTests(3)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle.AllTests(4)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestClassLifecycle.TheoryTestClassLifecycle.AllTests(5)<br clear="all"></div>
    </div>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class25')" ondblclick="ToggleClass('class25')"><span class="success">✔</span>
         tests.xunit.TestsInDerivedClasses.ConcreteBaseClass
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class25">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsInDerivedClasses.ConcreteBaseClass.BaseTestMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">0.001s</span><span class="clickable" onclick="ToggleClass('class26')" ondblclick="ToggleClass('class26')"><span class="success">✔</span>
         tests.xunit.TestsInDerivedClasses.DerivedFromAbstractBaseClass
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class26">
      <div class="row"><span class="timing">0.001s</span><span class="success">✔</span>
       tests.xunit.TestsInDerivedClasses.DerivedFromAbstractBaseClass.AbstractBaseTestMethod<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsInDerivedClasses.DerivedFromAbstractBaseClass.DerivedTestMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class27')" ondblclick="ToggleClass('class27')"><span class="success">✔</span>
         tests.xunit.TestsInDerivedClasses.DerivedFromConcreateBaseClass
         <span class="testcount">(2 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class27">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsInDerivedClasses.DerivedFromConcreateBaseClass.BaseTestMethod<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsInDerivedClasses.DerivedFromConcreateBaseClass.DerivedTestMethod<br clear="all"></div>
    </div>
    <h3><span class="timing">0.000s</span><span class="clickable" onclick="ToggleClass('class28')" ondblclick="ToggleClass('class28')"><span class="success">✔</span>
         tests.xunit.TestsInNestedClasses.ParentClass+NestedClass
         <span class="testcount">(1 test)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class28">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsInNestedClasses.ParentClass+NestedClass.NestedTestIsValid<br clear="all"></div>
    </div>
    <h3><span class="timing">0.007s</span><span class="clickable" onclick="ToggleClass('class29')" ondblclick="ToggleClass('class29')"><span class="success">✔</span>
         tests.xunit.TestsWithFixtures.TestsWithFixtures
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class29">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TestsWithFixtures.Test01<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TestsWithFixtures.Test02<br clear="all"></div>
      <div class="row"><span class="timing">0.002s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TestsWithFixtures.Test03<br clear="all"></div>
      <div class="altrow"><span class="timing">0.005s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TestsWithFixtures.Test04<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TestsWithFixtures.Test05<br clear="all"></div>
    </div>
    <h3><span class="timing">0.004s</span><span class="clickable" onclick="ToggleClass('class30')" ondblclick="ToggleClass('class30')"><span class="success">✔</span>
         tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures
         <span class="testcount">(5 tests)</span></span><br clear="all"></h3>
    <div class="indent" style="display: none;" id="class30">
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures.AllTests(1)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.004s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures.AllTests(2)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures.AllTests(3)<br clear="all"></div>
      <div class="altrow"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures.AllTests(4)<br clear="all"></div>
      <div class="row"><span class="timing">0.000s</span><span class="success">✔</span>
       tests.xunit.TestsWithFixtures.TheoryTestsWithFixtures.AllTests(5)<br clear="all"></div>
    </div><br><h5>Results generated 2009-06-18 at 22:34:49</h5>
  </body>
</html>